cnames_active.js

```diff
@@ -3351,6 +3351,7 @@ var cnames_active = {
   "yt-dislikes-viewer": "pgamerx.github.io/yt-dislikes-viewer",
   "yt-quota-manager": "arnav-kr.github.io/yt-quota-manager",
   "yu": "yuxizhe.github.io/yu",
+  "yuics": "cname.vercel-dns.com",  // noCF
```

Comment:
```suggestion
  "yuics": "cname.vercel-dns.com", // noCF
```
As the CI error notes, you've got an extra space here

Reply:
I'm sorry but it seems that js.org now do not support blog website. 😢 

PR Close now.